script.js

```diff
@@ -161,19 +162,11 @@ function QuickWiki() {
               var dy = d.offset().top;
               var xgap = e.pageX - dx;
               var ygap = e.pageY - dy;
-              if (!$.browser.msie) {
-                e.preventDefault();
```

Comment:
This line is still needed, right? The [documentation for `e.preventDefault()`][1] says that it prevents the default action of clicking the link, namely navigating to the new page.

[1]: http://api.jquery.com/event.preventdefault/